src/module_compilation.mli

```diff
@@ -2,33 +2,39 @@
 
 open Import
 
+module Includes : sig
```

Comment:
Is this module meant to be in the mli? It's not used outside Compilation_context.

Reply:
Indeed, in fact the implementation is no longer needed either as it was moved to `Compilation_context`. I removed it